include/c_types/path_rt.h

```diff
@@ -35,6 +35,8 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 #endif
 
 struct Path_rt {
+    /*Added route_id*/
```

Comment:
Modifying types will cause a lot of problems on the other functions.
I suggest you look how things are handled when there is an identifier of a route on the output.

Reply:
okay, that makes sense could you please tell me the name of the function that outputs path_id except pgr_ksp?